src/selectors/app.js

```diff
@@ -74,31 +88,39 @@ export const getTimelineHeight: Selector<null | CssPixels> = createSelector(
   getComputedHiddenGlobalTracks,
   getComputedHiddenLocalTracksByPid,
   getTrackThreadHeights,
+  getActiveTabGlobalTracks,
   getShowTabOnly,
+  getScreenshotTrackHeight,
   (
     globalTracks,
     localTracksByPid,
     hiddenGlobalTracks,
     hiddenLocalTracksByPid,
     trackThreadHeights,
-    showTabOnly
+    activeTabGlobalTracks,
+    showTabOnly,
+    screenshotTrackHeight
   ) => {
-    let height = TIMELINE_RULER_HEIGHT + TIMELINE_SETTINGS_HEIGHT;
+    let height = TIMELINE_RULER_HEIGHT;
```

Comment:
I wanted to separate this function as well but this is used by a root component(above the timeline) so it's not easy to switch selectors depending on the `showTabOnly`. I solved the active tab + full view in one function instead.